From 3e90b2992913db31bcc5b1b6f1b3fa8c28c5dc0c Mon Sep 17 00:00:00 2001
From: "Dmitry V. Levin" <ldv@altlinux.org>
Date: Thu, 9 Jun 2011 15:16:01 +0000
Subject: [PATCH] Fix MIPS syscall entries

* linux/mips/syscallent.h: Remove duplicate entries for 4336, 4337,
and 4338 syscall numbers.

Reported-by: Denys Vlasenko <dvlasenk@redhat.com>
---
 linux/mips/syscallent.h |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git linux/mips/syscallent.h linux/mips/syscallent.h
index 3382ffd..2306ab1 100644
--- linux/mips/syscallent.h
+++ linux/mips/syscallent.h
@@ -4682,9 +4682,6 @@
 	{ 5,	0,	printargs,		"o32_fanotify_mark"	}, /* 4337 */
 	{ 4,	0,	printargs,		"o32_prlimit64"	}, /* 4338 */
 #endif
-	{ 0,	0,	printargs,		"SYS_4336"	}, /* 4336 */
-	{ 0,	0,	printargs,		"SYS_4337"	}, /* 4337 */
-	{ 0,	0,	printargs,		"SYS_4338"	}, /* 4338 */
 	{ 0,	0,	printargs,		"SYS_4339"	}, /* 4339 */
 	{ 0,	0,	printargs,		"SYS_4340"	}, /* 4340 */
 	{ 0,	0,	printargs,		"SYS_4341"	}, /* 4341 */
-- 
1.7.5.4

